src/Roslyn.Diagnostics.Analyzers/CSharp/BlankLines/CSharpBlankLinesBetweenStatementsCodeFixProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable disable warnings
```

Comment:
Why not enable nullable warnings here?

Reply:
Copied from another file. I tried enabling this and it was pretty annoying.